scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+            grep "dev-" | tail -n +4 | awk '{print $1}' | xargs -r docker rmi || true
+        
+        # 시스템 정리
+        docker system prune -f
+        
+        echo "✅ Docker 정리 완료"
+EOF
+fi
+
+echo ""
+echo "✅ 개발 환경 리소스 정리가 완료되었습니다!"
+echo "💰 예상 비용 절약: 월 $10-15"
```

Shellcheck (0.10.0)

[error] Braces are required for positionals over 9, e.g. ${10}.

(SC1037)